CgfConverter/Services/BinaryReaderBigEndian.cs

```diff
@@ -1,56 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace CgfConverter.Services
-{
-    public class BinaryReaderBigEndian : BinaryReader
```

Comment:
Is this not needed for other console based Cryengine games?  Or did the functionality get moved somewhere else?

Reply:
I added EndianSwappableBinaryReader before to support big endianness all chunks, so this file has gone left unused.

Reply:
Ah, that's right.  I knew I had some tests for Sonic Boom and was confused how it would work without the endian reader.  👍🏽